config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  resources :users, only: [:index, :show] do 
+    resources :posts, only: [:index, :show]
+  end
+  root to: 'users#index'
+
+  get '/users/:id', to: 'users#show'
+  get '/users/:id/posts', to: 'posts#index'
+  get '/users/:id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] You have correctly configured the routes but I wanted to let you know that these are redundant and produces the same routes as the resources method. Hope it helps.